sensor/common/centralclient/grpc_connection.go

```diff
@@ -118,5 +123,5 @@ func (f *centralConnectionFactoryImpl) SetCentralConnectionWithRetries(conn *uti
 
 	conn.Set(centralConnection)
 	f.okSignal.Signal()
```

Comment:
This emits `SensorComponentEventCentralReachableHTTP` which is not fully correct, as the state is: "central reachable over HTTP" AND "gRPC connection set up but not ready yet".

However, I would like to keep it as it is for now, as in the code proposed in this PR, we stop relying on the `SensorComponentEventCentralReachableHTTP` signal.